src/OpenLoco/src/Audio/Audio.cpp

```diff
@@ -1208,6 +1229,119 @@ namespace OpenLoco::Audio
         return &kMusicInfo[track];
     }
 
+    static void sortPlaylistAlphabetically(std::vector<uint8_t>& tracks, bool reversed = false)
+    {
+        // Sort alphabetically using lambda expression that compares localised music titles
+        std::sort(tracks.begin(), tracks.end(), [reversed](int a, int b) {
+            const char* aTitle = StringManager::getString(kMusicInfo[a].titleId);
+            const char* bTitle = StringManager::getString(kMusicInfo[b].titleId);
+
+            // Compare the two strings alphabetically.
+            // Todo: make sure this is actually alphabetical no matter what localisation the player is using.
+            auto comparison = strcoll(aTitle, bTitle) < 0;
+
+            if (reversed)
+            {
+                return !comparison;
+            }
+            return comparison;
```

Comment:
this entire thing amounts to an XOR, so it is just
```
return !(strcoll(aTitle, bTitle) < 0) != !reversed;
```
though admittedly that is a little harder to read...